spec/models/user_spec.rb

```diff
@@ -14,4 +14,38 @@
     expect(user).to_not be_valid
     expect(user.errors[:password_confirmation]).to_not be_blank
   end
+
+  it 'should calculate the appropriate numbers' do
+    user = User.create!(username: "u", email: "u@u.com", password: "12345678")
+    user2 = User.create!(username: "t", email: "t@u.com", password: "12345678")
+    user3 = User.create!(username: "d", email: "d@u.com", password: "12345678")
+
+    convo = Conversation.create!(sender: user, recipient: user2)
+    rating = Rating.create!(rater: user, ratee: user2, conversation: convo, score: 10)
+    rating2 = Rating.create!(rater: user2, ratee: user, conversation: convo, score: 1)
```

Comment:
Useless assignment to variable - `rating2`. Did you mean `rating`?<br>Line is too long. [86/80]